core/src/Borrower.sol

```diff
@@ -295,21 +295,19 @@ contract Borrower is IUniswapV3MintCallback {
         int24[] memory positions_ = positions.write(callee.callback(data, msg.sender));
         _saveSlot0(uint160(msg.sender), _formatted(State.Ready));
 
-        (uint256 ante, uint256 nSigma, uint256 pausedUntilTime) = FACTORY.getParameters(UNISWAP_POOL);
-
-        (Prices memory prices, bool isSus) = _getPrices(nSigma, oracleSeed);
-        Assets memory assets = _getAssets(positions_, prices, false);
         (uint256 liabilities0, uint256 liabilities1) = _getLiabilities();
+        if (liabilities0 > 0 || liabilities1 > 0) {
+            (uint256 ante, uint256 nSigma, uint256 pausedUntilTime) = FACTORY.getParameters(UNISWAP_POOL);
+            (Prices memory prices, bool seemsLegit) = _getPrices(oracleSeed, nSigma);
 
-        require(BalanceSheet.isHealthy(prices, assets, liabilities0, liabilities1), "Aloe: unhealthy");
-        unchecked {
-            if (liabilities0 + liabilities1 > 0)
-                require(
-                    address(this).balance > ante && !isSus && block.timestamp > pausedUntilTime,
-                    "Aloe: missing ante / sus price"
-                );
+            require(
+                seemsLegit && (block.timestamp > pausedUntilTime) && (address(this).balance >= ante),
```

Comment:
Why go from using gt to using gte?

Reply:
So that we don't have to add the 1 wei at the end. Nbd in frontend code, but if someone ever wants to fund a `Borrower` manually, it's much easier to type "0.01" than "0.010000000...00001"